server/hap2/hatohol/hap2_nagios_ndoutils.py

```diff
@@ -182,12 +194,19 @@ def collect_triggers_and_put(self, fetch_id=None, host_ids=None):
             in_cond = "','".join(host_ids)
             sql += " WHERE %s.host_object_id in ('%s')" % (t2, in_cond)
 
-        # NOTE: The update time in the output is renewed every status
-        #       check in Nagios even if the value is not changed.
-        # TODO by 15.09:
-        #   We should has the previous result and compare it here in order to
-        #   improve performance. Or other columns such as last_state_change in
-        #   nagios_servicestatus might be used.
+        def all_triggers_should_send():
```

Comment:
This tiny function may be simpler when we use lambda.
<pre>
all_triggers_should_send = lambda: fetch_id is None
</pre>